SignalMessaging/utils/OWSAudioPlayer.m

```diff
@@ -99,11 +99,20 @@ - (void)playWithCurrentAudioCategory
 - (void)playWithPlaybackAudioCategory
 {
     OWSAssertIsOnMainThread();
-    [OWSAudioSession.shared setPlaybackCategoryWithAudioActivity:self.audioActivity];
+    [OWSAudioSession.shared startPlaybackAudioActivity:self.audioActivity];
 
     [self play];
 }
 
+- (void)playAsForegroundAlert
+{
+    OWSAssertIsOnMainThread();
+    [OWSAudioSession.shared startAmbientAudioActivity:self.audioActivity];
+
+    AudioServicesPlaySystemSound(kSystemSoundID_Vibrate);
```

Comment:
Replace the haptic feedback which we'd lost (inadvertently I believe) since 2.20